tests/test_config_consistency.py

```diff
@@ -555,19 +557,59 @@ def test_log_injection_128bit_traceid_disabled(self):
 @scenarios.runtime_metrics_enabled
 @features.tracing_configuration_consistency
 class Test_Config_RuntimeMetrics_Enabled:
-    """Verify runtime metrics are enabled when DD_RUNTIME_METRICS_ENABLED=true"""
+    """Verify runtime metrics are enabled when DD_RUNTIME_METRICS_ENABLED=true and that they have the proper tags"""
 
-    # This test verifies runtime metrics by asserting the prescene of a metric in the dogstatsd endpoint
-    def test_config_runtimemetrics_enabled(self):
-        for data in interfaces.library.get_data("/dogstatsd/v2/proxy"):
-            lines = data["request"]["content"].split("\n")
-            metric_found = False
-            for line in lines:
-                if runtime_metrics[context.library.library] in line:
-                    metric_found = True
-                    break
-            assert metric_found, f"The metric {runtime_metrics[context.library.library]} was not found in any line"
-            break
+    def setup_main(self):
+        self.req = weblog.get("/")
+
+    def test_main(self):
+        assert self.req.status_code == 200
+
+        time.sleep(12)
```

Comment:
There is no reason to have a `sleep` here, at this point, the infra is down and all data are available. Could you tell me more why you add it ?

Reply:
I thought I left a comment, sorry about that. The default interval for for Runtime Metrics to be submitted from the tracer is 10 seconds. Rather than trying to add a config to tweak this, I'd like to run a small number of tests (in this case 2) that wait for the default interval.

I decided to sleep 12 seconds but perhaps 10 seconds would be fine since there's container startup, a healthcheck, and the setup code that's already been run on the app?

Reply:
The point is that during any `test_` method, the infra is shutdown since a while, so it's pointless to wait here.

If any sleep may be used, it's in the `setup` method.